packages/bazel/src/schematics/sync/files/build-file/BUILD.bazel__tmpl__

```diff
@@ -0,0 +1,39 @@
+# ==== Generated by Nx
+# ==== Do not touch
+
+load("@npm//@nrwl/cli:index.bzl", "nx", "nx_test")
+
+filegroup(
+    name = "<%= project.name %>",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+nx(
+    name = "build",
+    args = [
+        "build",
+        "<%= project.name%>",
+        "--outputPath=$@",
+    ],
+    data = [
+        ":<%= project.name %>",
+        # Dependencies
+        <% for (dep of depGraph.dependencies[project.name]) { %>"//<%= depGraph.projects[dep.projectName].root %>:<%= dep.projectName %>",
+        <% } %>
+        # Root Files
+        "//:angular.json",
+        "//:package.json",
+        "//:tsconfig.json",
+        # NOTE: Use @npm//node_modules causes bundling to fail due to Babel errors.
```

Comment:
I thought that has been fixed. I'd prefer not to list individual packages.

Reply:
@mrmeku has a fix for this.